tests/autorestart/test_container_autorestart.py

```diff
@@ -79,6 +79,8 @@ def ignore_expected_loganalyzer_exception(duthosts, enum_dut_feature_container,
             ".*ERR teamd[0-9]*#teamsyncd.*readData.*netlink reports an error=-33 on reading a netlink socket.*",
             ".*ERR swss[0-9]*#orchagent.*set status: SAI_STATUS_ATTR_NOT_IMPLEMENTED_0.*",
             ".*ERR swss[0-9]*#orchagent.*setIntfVlanFloodType.*",
+            ".*ERR swss[0-9]*#buffermgrd.*Failed to process invalid entry.*",
+            ".*ERR swss[0-9]*#buffermgrd.*Unable to create/update PG profile.*",
```

Comment:
Can be removed. This is already covered in the common ignore list 
https://github.com/Azure/sonic-mgmt/blob/master/ansible/roles/test/files/tools/loganalyzer/loganalyzer_common_ignore.txt#L20

Reply:
@neethajohn Can we also put the regex at line 82 in the common ignore file?

Reply:
@neethajohn Removed.